resources/standards-data/ldml-keyboards/techpreview/fetch-latest-cldr-techpreview.sh

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env bash
+
+# Copyright:    © SIL International.
+# Description:  Import new files from CLDR tech preview
+# Create Date:  17 Oct 2022
+# Authors:      Steven R. Loomis (SRL)
+
+if [[ $# -ne 1 ]];
+then
+    echo >&2 "Usage: $0 <cldr-dir>"
+    exit 1
+fi
+
+CLDR_DIR="$1"
+shift
+
+KEYBOARDS_DIR="${CLDR_DIR}/keyboards"
+DTD_DIR="${KEYBOARDS_DIR}/dtd"
+IMPORT_DIR="${KEYBOARDS_DIR}/import"
+DATA_DIR="${KEYBOARDS_DIR}/3.0"
+TEST_DIR="${KEYBOARDS_DIR}/test"
+
+# a file to check
+CHECK_1="${DTD_DIR}/ldmlKeyboard.dtd"
+CHECK_2="${DTD_DIR}/ldmlKeyboardTest.dtd"
+
+if [[ ! -f "${CHECK_1}" ]];
+then
+    echo >&2 "$0: error: ${CHECK_1} did not exist"
+    echo >&2 "$0: error: Is ${CLDR_DIR} a valid cldr keyboard directory?"
+    exit 1
```

Comment:
Could use `die()` from build-utils.sh instead with a two-line message?

Reply:
using die, but how do I get a two line message to `die()`?